### PR TITLE
Fix PCT failures past 2.281

### PR DESCRIPTION
**Pull request by res0nance**
* JENKINS issue(s):
    * N/A
* Description:
    * Fixed PCT failures see jenkinsci/bom#511
* Documentation changes:
    * No docs needed
* Users/aliases to notify:
    * @bitwiseman

### CODE REVIEW

**Review by bitwiseman — APPROVED** [review]

File: pipeline-model-definition/src/test/java/org/jenkinsci/plugins/pipeline/modeldefinition/generator/DirectiveGeneratorTest.java [hunk 1]
```diff
@@ -150,12 +152,12 @@ public void simpleInput() throws Exception {
 
     // TODO: Remove once we move to a baseline of 2.90 or later
```
Comment (bitwiseman) [inline pipeline-model-definition/src/test/java/org/jenkinsci/plugins/pipeline/modeldefinition/generator/DirectiveGeneratorTest.java:153]: @res0nance 
We've moved to a baseline way past 2.90.  Should this be removed?

**Review by bitwiseman — CHANGES_REQUESTED** [review]
Sorry, one question.

File: pipeline-model-definition/src/test/java/org/jenkinsci/plugins/pipeline/modeldefinition/generator/DirectiveGeneratorTest.java (hunk 1 above)
Comment (res0nance) [reply pipeline-model-definition/src/test/java/org/jenkinsci/plugins/pipeline/modeldefinition/generator/DirectiveGeneratorTest.java:153]: Let me reword it

**Comment by jetersen** [comment]
Seems the build failure is related to infra issues, if someone could trigger the build that would be nice :)

**Review by bitwiseman — APPROVED** [review]

**Comment by jetersen** [comment]
Another infra issue it seems:
```
hudson.plugins.git.GitException: Command "git.exe fetch --tags --force --progress -- C:\Users\jenkins\Work\workspace\e-model-definition-plugin_PR-419\pipeline-model-definition\target\tmp\junit3356454706115552377\junit5235385210740340602 +refs/heads/*:refs/remotes/origin/*" returned status code 128:
stdout: 
stderr: error: git upload-pack: git-pack-objects died with error.
fatal: git upload-pack: aborting due to possible repository corruption on the remote side.
remote: aborting due to possible repository corruption on the remote side.&#0;
fatal: protocol error: bad pack header
```

**Comment by jetersen** [comment]
Poke: This passes.
A release would be nice :)

**Comment by jglick** [comment]
@bitwiseman ping

**Comment by timja** [comment]
@bitwiseman @car-roll @dwnusbaum can we get a release please?

**Comment by bitwiseman** [comment]
Will do.

**Comment by saper** [comment]
Hello, I came here from the release notes - what exactly is PCT?

**Comment by timja** [comment]
A tool for testing plugin compatibility against other plugins and newer Jenkins core version, a compatibility break in tests was picked up from a Jenkins core change which required test fixes here.

https://github.com/jenkinsci/plugin-compat-tester

**Comment by jetersen** [comment]
A search on jenkins-ci org reveals this: https://github.com/search?q=org%3Ajenkinsci+pct&type=source

**Pull request merged**